webapp-django/crashstats/signature/jinja2/signature/signature_report.html

```diff
@@ -13,21 +13,28 @@
 {% block content %}
 <div id="mainbody">
     <div class="page-heading">
+        <a href="#" class="display-toggle-filters toggle-filters show">Show Search Filters</a>
```

Comment:
Also, I wonder about calling it "Search". **I** know it's a search because the underlying tech is SuperSearch. But the page is, after all, called "Signature report ..."
So perhaps "Show Filters" (and "Hide Filters").